src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,115 @@
+.card {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+
+  background-color: rgb(255, 255, 255);
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+
+  font-family: Roboto, sans-serif;
+
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+
+    display: block;
+    margin: 0 auto;
+  }
+
+  &__title {
+    width: 100%;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 4px;
+
+    color: #060b35;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__text-code {
+    margin: 0;
+    font-size: 10px;
+    color: #616070;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__text-review {
+    margin: 0;
+    font-size: 10px;
+    color: #060b35;
+    font-weight: 400;
+  }
+
+  &__review {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-top: 16px;
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-top: 24px;
+    margin-bottom: 16px;
+  }
+
+  &__text-price {
+    font-size: 12px;
+    color: #616070;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__text-value {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button {
+    width: 100%;
+    height: 40px;
+    margin: 0;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-size: 14px;
+    font-weight: 700;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    color: white;
+    background-color: $active-color;
```

Comment:
The variable `$active-color` is used for styling the button's background color. Ensure that this variable is defined in your SCSS files, otherwise it will cause a compilation error.